### ISSUE TITLE
Outdated CeCILL_LICENSE

### ISSUE THREAD

**Issue by mwuttke97** [uds-psl/coq-library-undecidability#38]
This software is licensed under the `CeCILL FREE SOFTWARE LICENSE AGREEMENT`, `Version 2.0 dated 2006-09-05`.

This version is not affirmed by the OSI. However, updating to version 2.1 should be possible due to point 12.3 of the agreement.

Background (wikipedia):

> Version 2.1 was released in June 2013. It allows relicensing to the GNU Affero General Public License and the European Union Public License as well as the GPL, and clarifies the language that requires licensees to give access to the source code (which had previously caused rejection of version 2.0 by the Open Source Initiative[3]).

**Comment by yforster** [comment]
I'm happy to change to the 2.1 version immediately. @DmxLarchey do you agree?

**Comment by DmxLarchey** [comment]
Yep I agree, You can get the text file of the `V2.1` of the licence from there:

https://cecill.info/licences/Licence_CeCILL_V2.1-en.txt
